sssd_test_framework/utils/sssctl.py

```diff
@@ -188,6 +188,29 @@ def user_checks(self, username: str, action: str = "acct", service: str = "syste
         """
         return self.host.ssh.exec(["sssctl", "user-checks", username, "-a", action, "-s", service])
 
+    def user_show(self, user: str | None = None, sid: str | None = None, uid: str | None = None) -> SSHProcessResult:
```

Comment:
I would also accept uid as int.

Reply:
That makes sense. Probably SID can be int as well.

Reply:
I don't think SID can be an `int` as well as it contains letters and hyphens. Example:
```
S-1-5-21-1004336348-1177238915-682003330-512
```

Reply:
Oh I see. I will update it. Thanks for info